nginx/Dockerfile

```diff
@@ -39,9 +39,18 @@ RUN --mount=type=cache,id=nginx-apk,sharing=locked,from=cache,target=/var/cache/
     cleanup.sh
 
 ENV \
+    NGINX_CLIENT_BODY_TIMEOUT=60 \
     NGINX_CLIENT_MAX_BODY_SIZE=0 \
     NGINX_ERROR_LOG_LEVEL=warn \
+    NGINX_FASTCGI_READ_TIMEOUT=60 \
+    NGINX_FASTCGI_SEND_TIMEOUT=60 \
+    NGINX_FASTCGI_CONNECT_TIMEOUT=60 \
     NGINX_KEEPALIVE_TIMEOUT=65 \
```

Comment:
Just noticed this isn't the same as the default: https://nginx.org/en/docs/http/ngx_http_core_module.html#keepalive_timeout